react-vite/base/src/theme/variables.css

```diff
@@ -234,3 +234,9 @@ http://ionicframework.com/docs/theming/ */
     --ion-card-background: #1e1e1e;
   }
 }
+
+html {
+  /* For more information on dynamic font scaling, visit the documentation: 
```

Comment:
Is there a reason this comment is formatted differently?

Reply:
CSS vs. SCSS.

I tried to match the existing comment block syntax for each file.

Reply:
Oh I see, okay sounds good